src/main/java/org/spdx/library/model/ModelObject.java

```diff
@@ -1002,8 +1002,20 @@ public boolean equals(Object o) {
 	 * @return
 	 */
 	public ModelObject clone(IModelStore modelStore) {
+		if (Objects.isNull(this.copyManager)) {
+			throw new RuntimeException("A copy manager must be provided to clone");
```

Comment:
nit: Maybe `IllegalStateException` would be a bit more precise?

Reply:
Agree - changed to `IllegalStateException`